rust/actyx/ax-types/src/lib.rs

```diff
@@ -0,0 +1,38 @@
+#![doc(html_logo_url = "https://developer.actyx.com/img/logo.svg")]
+#![doc(html_favicon_url = "https://developer.actyx.com/img/favicon.ico")]
+#![allow(clippy::unreadable_literal)]
+#![allow(clippy::inconsistent_digit_grouping)]
+
+#[doc(hidden)]
+pub use ax_macros::*;
+
+#[macro_use]
+mod scalar;
+
+mod app_manifest;
+mod event;
+mod offset;
+mod scalars;
+pub mod service;
+mod tags;
+mod timestamp;
+pub mod types;
+
+pub use app_manifest::AppManifest;
+pub use event::{Event, EventKey, Metadata, Opaque, Payload};
+pub use offset::{Offset, OffsetError, OffsetMap, OffsetOrMin};
+pub use scalars::{AppId, NodeId, StreamId, StreamNr};
+pub use tags::{Tag, TagSet};
+pub use timestamp::{LamportTimestamp, Timestamp};
+
+use derive_more::Display;
```

Comment:
I prefer the fully-qualified name inside `derive` because it doesn’t pollute the scope